core/src/test/scala/org/apache/spark/deploy/StandaloneDynamicAllocationSuite.scala

```diff
@@ -395,8 +395,8 @@ class StandaloneDynamicAllocationSuite
     }
 
     var apps = getApplications()
-    // kill executor 1
-    assert(sc.killExecutor(executors.head))
+    // kill executor 1, and actually nothing to kill
+    assert(!sc.killExecutor(executors.head))
```

Comment:
I think this may be a bigger problem. This is supposed to be `true` because it's a new executor so we should be able to kill it. I think it's the way this test is set up that makes this false, because the driver doesn't wait for new executors to come up. We might need to do some more mocking here.